utils/getRandomArrayIndex.ts

```diff
@@ -0,0 +1,3 @@
+export const getRandomArrayIndex = (array: string[]) => {
```

Comment:
It doesn't have to be string, it will be working for any elements.
```suggestion
export const getRandomArrayIndex = <T>(array: T[]) => {
```